src/sentry/static/sentry/app/views/organizationIntegrations/integrationCodeMappings.tsx

```diff
@@ -76,7 +76,7 @@ class IntegrationCodeMappings extends AsyncComponent<Props, State> {
     return [
       [
         'pathConfigs',
-        `/organizations/${orgSlug}/integrations/${this.integrationId}/repo-project-path-configs/`,
+        `/organizations/${orgSlug}/repo-project-path-configs/?integrationId=${this.integrationId}`,
```

Comment:
@NisanthanNanthakumar you should specify query parameters as the third argument (see `repos` below).